After cr (#3)
onetads/wordpress-plugin: readme.txt

=== Retail Media Network - Onet Ads ===

> Contributors: Ringier Axel Springer Polska sp. z o.o.

Reviewer: > The following contributors listed were ignored, as the WordPress.org user could not be found. Ringier Axel Springer Polska sp. z o.o.. The Contributors field should only contain WordPress.org usernames.

https://wordpress.org/plugins/developers/readme-validator/

> Requires at least: 4.7

Reviewer: ```suggestion
Tags: woocommerce, ecommerce, ads, shop, retail
Requires at least: 4.7
```

Tested up to: 6.5.2
Requires PHP: 7.3
Stable tag: 1.0

License: GPLv3
License URI: https://www.gnu.org/licenses/gpl-3.0.html

> Rapid integration of Woocommerce with Retail Media Network in Onet Ads.

Reviewer: ```suggestion
Instant integration of WooCommerce with Retail Media Network in Onet Ads. Use this plugin to quickly and intuitively implement our advertising formats in your store.
```

> == Requirements ==

Reviewer: ```suggestion
== Description ==

Instant integration of WooCommerce with Retail Media Network in Onet Ads. Use this plugin to quickly and intuitively implement our advertising formats in your store.

= Driving Sales in the Best Stores =

Retail Media Network by Onet Ads is a dynamically developing advertising network that connects online stores with manufacturers.

[Join RMN - register with Onet Ads](https://panel.onetads.pl/)

= Why is it worth it? =

We create innovative, pioneering advertising services on the Polish market based on the ad inventory of the best retailers, strengthened by the power of Onet and RAS Polska services. Check out how it works:

= Earn extra on your store’s inventory =

Retail Media Network is a new revenue channel for your business. Being part of the network will provide you with additional budgets from advertisers. Importantly, traffic from ad campaigns stays in your store.

= Gain valuable traffic from RASP inventory =

Retail Media Network is the synergy of advertising inventory of stores and RASP. In our services, we display advertising formats that direct users to stores present in the network. For you, this is a source of valuable traffic.

= Use the modern Onet Ads panel =

You will manage your presence and activity in the Retail Media Network using Onet Ads. It is a friendly and intuitive self-service panel. We provide a transparent ecosystem, standardized workflow, clear cooperation rules, and ongoing support whenever you need it.

= Discover the benefits of RMN with free integration =

Integration with Retail Media Network is free, and we do not require any form of business exclusivity. For the purposes of integration, we will provide you with the necessary documentation and expert support.

= Join the Retail Media Network in Onet Ads today =

[Go to the Onet Ads panel](https://panel.onetads.pl/)

== Requirements ==
```

    WordPress version 6.5.2 or newer
    PHP version 7.3 or newer

== Installation ==

= Searching for the Plugin =

    Go to “Plugins” in the WordPress admin panel, then select “Add New.”
    Type “Retail Media Network - Onet Ads” into the search field and find the plugin developed
    by Ringier Axel Springer Polska sp. z o.o.
    Pick the right plugin, called “Retail Media Network - Onet Ads,” and hit “Install.”
    Activate the plugin once the installation is complete.

= Downloading and Uploading the Plugin =

    Get the “Retail Media Network - Onet Ads” plugin from the official WordPress website.
    In the WordPress admin panel, navigate to “Plugins” and select “Add New.”
    Click on “Upload Plugin,” choose the zip file you’ve downloaded, and press “Install Now”.
    Activate the plugin after it’s installed.

== Usage ==

    After installing the plugin, just turn it on.

== License ==

> This project is licensed under the GNU General Public License v2.0 or later. Details can be found in the LICENSE file.

Reviewer: ```suggestion
This project is licensed under the GNU General Public License v3.0 or later. Details can be found in the LICENSE file.
```
